backend/covexit/account/models.py

```diff
@@ -1,7 +1,43 @@
+import random
+import string
+
 from django.contrib.auth.models import User
 from django.db import models
 from django.utils.translation import ugettext_lazy as _
 from django.core.exceptions import ValidationError
+from django.core.mail import send_mail
+from django.conf import settings
+
+VERIFICATION_MESSAGE = """Thank you for signing up for Covexit!
+
+To activate your account, please visit this link:
+{}
+"""
+
+VERIFICATION_KEY_LENGTH = 30
+# Note: this will have to be set up on the frontend:
+VERIFICATION_URL = 'https://covexit.de/verify/'
+
+
+def create_verification_link(user):
+    return VERIFICATION_URL + user.profile.verification_key
+
+
+def create_verification_key():
+    """Return a random string."""
+    return ''.join(random.choices(string.ascii_letters,
+                                  k=VERIFICATION_KEY_LENGTH))
+
+
+def send_verification_email(user):
+    link = create_verification_link(user)
+    send_mail(
+        'Covexit Email Verification',
+        VERIFICATION_MESSAGE.format(link),
+        settings.DEFAULT_FROM_EMAIL,
+        [user.email],
+        fail_silently=False,
+    )
```

Comment:
Should we do a proxy model to User so we have this all directly on the User instance? Or does it not work that way?